src/components/AccountLoginForm.jsx

```diff
@@ -1,241 +1,25 @@
-
-      // Build common fields
-      const hydrate = hydrateFieldValue[name] || identity
-      const attributes = {
-        ...field, // TODO be more restrictive on what comes in
-        disabled: disabled || readonly,
-        invalid: !!error,
-        onEnterKey,
-        giveFocus,
-        label: t(`${konnectorSlug}.fields.${field.name}.label`, {
-          _: t(`account.form.label.${label || name}`)
-        }),
-        readonly: readonly,
-        value: isUnloading ? '' : hydrate(value)
-      }
-
-      if (placeholder) {
-        attributes.placeholder = placeholder
-      }
-
-      return (
-        <div className={fieldStyles['coz-field-wrapper']}>
-          <FieldDescription field={field} konnectorSlug={connectorSlug} t={t} />
-          {React.cloneElement(renderers[type](this.props), attributes)}
-        </div>
-      )
-    }
-
-    return (
-      // We use a <div> instead of a <form> to disable the "use password for" function of Chrome
-      <div className={styles['account-form-login']} role="form">
-        {/* Error */}
-        {error && (
-          <p className="errors">{t('account.message.error.bad_credentials')}</p>
-        )}
-
-        {/* Fields */}
-        {!!editableFields &&
-          !!editableFields.length && (
-            <fieldset className={styles['account-form-fieldset']}>
-              {editableFields.map(renderField(connectorSlug))}
-            </fieldset>
-          )}
-        {editing &&
-          !displayAdvanced &&
-          !!advancedFields.length && (
-            <button
-              type="button"
-              className={styles['col-account-form-advanced-button']}
-              onClick={toggleAdvanced}
-            >
-              {t('account.form.button.advanced')}
-            </button>
-          )}
-
-        {editing &&
-          displayAdvanced &&
-          !!advancedFields &&
-          !!advancedFields.length && (
-            <fieldset className={styles['account-form-fieldset']}>
-              {advancedFields.map(renderField(connectorSlug))}
-            </fieldset>
-          )}
-
-        {/* Controls */}
-        <div className={styles['coz-form-controls']}>
-          {(!editing || hasEditableFields) &&
-            !isSuccess && (
-              <Button
-                className={styles['coz-btn']}
-                disabled={oAuthTerminated || submitting || !submitEnabled}
-                busy={
-                  submitting &&
-                  !disableSuccessTimeout &&
-                  (editing || !isOAuth || oAuthTerminated)
-                }
-                onClick={onSubmit}
-              >
-                {t(
-                  editing
-                    ? 'account.form.button.save'
-                    : 'account.form.button.connect'
-                )}
-              </Button>
-            )}
-        </div>
-      </div>
+    const { features } = this.context
+    const { account, konnector } = this.props
```

Comment:
Maybe extract and transmit `this.props.editing` , we will use it in Harvest ;)

Reply:
As seen with @sizvix, we'll use `initialValues` props instead of passing an account.